server/controllers/authentication.js

```diff
@@ -0,0 +1,35 @@
+import User from '../models/userModel';
+import { buildResponse } from '../utils/helpers';
+
+exports.signUp = (req, res, next) => {
+  const { name, email, password } = req.body;
+  if (!email || !password) {
+    const errorMessage = 'You must provide email and password';
+    const response = buildResponse(false, errorMessage);
+    return res.status(400).send(response);
+  }
+  User.findOne({ email }, (err, existingUser) => {
+    if (err) {
+      return next(err);
+    }
+    if (existingUser) {
+      const errorMessage = 'Email already in use';
+      const response = buildResponse(false, errorMessage);
+      return res.status(400).send(response);
+    }
+    const user = new User({
+      name,
+      email,
```

Comment:
is email case sensitive? if yes please make email case insensitive

Reply:
I am making email to lowercase before saving and searching.